ZAPD/ZDisplayList.cpp

```diff
@@ -2294,7 +2294,7 @@ std::string ZDisplayList::GetExternalExtension()
 
 std::string ZDisplayList::GetSourceTypeName()
 {
-	return "Gfx";
+	return isStatic ? "static Gfx" : "Gfx";
```

Comment:
This part seems a bit unnecessary , because ZFile seems to handle the `declType` by itself, right?

Reply:
Good point. ZFile might be able to handle this itself since it knows what ZResources are flagged as static.